relays/ethereum/src/main.rs

```diff
@@ -41,12 +42,14 @@ mod utils;
 
 use ethereum_client::{EthereumConnectionParams, EthereumSigningParams};
 use ethereum_sync_loop::EthereumSyncParams;
+use instances::{BridgeInstance, KovanInstance, RialtoInstance};
 use parity_crypto::publickey::{KeyPair, Secret};
 use sp_core::crypto::Pair;
-use std::io::Write;
 use substrate_client::{SubstrateConnectionParams, SubstrateSigningParams};
 use substrate_sync_loop::SubstrateSyncParams;
 
+use std::io::Write;
```

Comment:
I prefer a single (but sorted) group of imports. IMHO splitting them arbitrarily never works long-term in a bigger project.

Reply:
I've been splitting imports like this:

1. Crate imports
2. Crates.io imports
3. Std imports

I'd be okay merging 2 and 3, but I'd like to keep the crate imports separate since it gives me a way to tell what we're putting in from the rest of the project at a glance

Reply:
I've noticed that it's quite common in other files already, so I'm okay with that. The issue I have with `crate` stuff being in separate section is that I've seen in all of our other "big" projects that at some point some stuff is moved to a separate crate, and the imports are usually changed by search & replace (then format to sort) and over time if we don't pay enough attention we might end up with two sections, but the original idea will be lost. Hope it's not going to happen here though :)